packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -72,6 +73,14 @@ const validProjectName = (s) => {
     return regex.test(s) || 'Value can only contain letters, numbers, space and hyphens.'
 }
 
+const validAppExtensionNameRegex = nameRegex
```

Comment:
Could we simply have just assigned that regex to "validAppExtensionNameRegex" from the start?